.github/workflows/darker.yml

```diff
@@ -0,0 +1,21 @@
+name: 🧹 Darker
```

Comment:
We can just use black - we use darker on `nucypher` so that we don't end up with one massive formatting commit

Reply:
✔️